installer/roles/image_build/templates/Dockerfile.j2

```diff
@@ -140,12 +138,17 @@ RUN dnf -y update && \
     vim-minimal \
     which \
     xmlsec1-openssl && \
-    dnf -y --repofrompath gcloud,https://packages.cloud.google.com/yum/repos/cloud-sdk-el8-x86_64 \
-       --setopt gcloud.gpgkey=https://packages.cloud.google.com/yum/doc/rpm-package-key.gpg \
-      install kubectl && \
     dnf -y install centos-release-stream && dnf -y install "rsyslog >= 8.1911.0" && dnf -y remove centos-release-stream && \
     dnf -y clean all
 
+# Install kubectl
+RUN curl -L -o /usr/bin/kubectl https://storage.googleapis.com/kubernetes-release/release/v1.17.8/bin/linux/{{ kubectl_architecture | default('amd64') }}/kubectl && \
```

Comment:
Is it possible to do this in a way that doesnt require pinning the version of kubectl?

Reply:
The k8s docs[1] show using a command like the following, but I personally don't like that for various reasons.

```
curl -LO "https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/linux/amd64/kubectl"
```

[1] https://kubernetes.io/docs/tasks/tools/install-kubectl/